index.js

```diff
@@ -1 +1,40 @@
-console.log('Happy developing ✨')
+const operationButtons = document.querySelectorAll('.operation-button');
+const equalButton = document.querySelector('#button-equal');
+const clearButton  = document.querySelector('#button-clear');
+const calculatorScreen = document.querySelector('#screen');
+const errorMessage = document.querySelector('.error-message');
+
+operationButtons.forEach(function(operationButton) {
+  operationButton.addEventListener('click', function() {
+    let operationValue = operationButton.innerText;
```

Comment:
`let` is not necessary here

Reply:
But you have to use either `let` or `const`. If `let` is not necessary, use `const`